src/main/java/org/helpberkeley/memberdata/User.java

```diff
@@ -1175,84 +1175,239 @@ static User createUser(
         return user;
     }
 
+    public enum ReportHeaderOption {
+        ADD_EMAIL,
+        NO_EMAIL
+    }
+
```

Comment:
Thank you for adding this.  The enum style in this project, is either to define the enum in its own file, or to move the definition to the top of the class where it is defined, before the data members.  I will move this as part my next commit in this branch.